### PR TITLE
update gc can't work alert

### LINKED ISSUE #9910: the rule for TiKV_GC_can_not_work alert should be enhanced to check with GC speed instead of GC task
## Bug Report

ctive Since	Value
alertname="TiKV_GC_can_not_work" env="ENV_LABELS_ENV" expr="sum(increase(tikv_gcworker_gc_tasks_vec{task="gc"}[1d])) < 1" level="emergency"	FIRING	2021-03-27 15:10:06.568417073 +0000 UTC

sometimes, the system maybe has not update or delete record, should not triggered this alert.

### What version of TiKV are you using?
v5.0.0

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve?

Make alert manager check gc work status correctly.

Issue Number: close #9910 .

### Check List 

Tests 

- No code

Side effects

### Release note 
* No release note.

### CODE REVIEW

**Review by hunterlxt — APPROVED** [review]
lgtm

File: metrics/alertmanager/tikv.rules.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ groups:
       summary: TiKV memory used too fast
 
   - alert: TiKV_GC_can_not_work
-    expr: sum(increase(tikv_gcworker_gc_tasks_vec{task="gc"}[1d])) < 1
+    expr: sum(increase(tikv_storage_mvcc_gc_delete_versions_sum[1d])) < 1 and (sum(increase(tikv_gcworker_gc_tasks_vec{task="gc"}[1d])) < 1 or sum(increase(tikv_gc_compaction_filter_skip[1d])) < 1)
```
Comment (youjiali1995) [inline metrics/alertmanager/tikv.rules.yml:17]: Please add parentheses.

Comment (MyonKeminta) [reply metrics/alertmanager/tikv.rules.yml:17]: I've confirmed with @hicqu that the expression is not correct. I think he will update it.

**Review by MyonKeminta — APPROVED** [review]
LGTM

**Pull request merged**